sda-commons-dependencies/build.gradle

```diff
@@ -197,12 +197,15 @@ dependencies {
     // sda-commons-server-opentelemetry
     api "io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:${openTelemetryVersion}"
     api "io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-apache-httpclient-5.0:${openTelemetryVersion}-alpha"
+    api "io.opentelemetry.instrumentation:opentelemetry-apache-httpclient-5.2:2.1.0-alpha"
     api "io.opentelemetry:opentelemetry-exporter-otlp:$openTelemetryVersion"
     api "io.opentelemetry:opentelemetry-exporter-jaeger:$openTelemetryVersion"
     api "io.opentelemetry:opentelemetry-extension-trace-propagators:$openTelemetryVersion"
     api "io.opentelemetry:opentelemetry-sdk-testing:$openTelemetryVersion"
     api "io.opentelemetry:opentelemetry-exporter-logging:$openTelemetryVersion"
-    //api "io.opentelemetry:opentelemetry-semconv:${openTelemetryVersion}"
+    api "io.opentelemetry:opentelemetry-semconv:1.30.1-alpha"
+    api "io.opentelemetry.semconv:opentelemetry-semconv:1.23.1-alpha"
+    api "io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:2.1.0"
```

Comment:
We should create a follow up to cleanup the OTEL dependencies. We enforce the OTEL-bom (why do we need two, do they conflict?) but still have to manage a lot on our own. I hope that can be simplified.